addons/crm/report/crm_activity_report.py

```diff
@@ -12,34 +12,41 @@ class ActivityReport(models.Model):
     _description = "CRM Activity Analysis"
     _rec_name = 'id'
 
-    date = fields.Datetime('Date', readonly=True)
-    author_id = fields.Many2one('res.partner', 'Created By', readonly=True)
+    date = fields.Datetime('Completition Date', readonly=True)
+    create_date = fields.Datetime('Creation Date', readonly=True)
```

Comment:
Rename to Lead Creation Date ? Otherwise it is not clear if the creation is the one from activity or lead. Other fields are more self explanatory I think, but not this one.